python/pyspark/errors/exceptions/captured.py

```diff
@@ -104,7 +104,7 @@ def getMessageParameters(self) -> Optional[Dict[str, str]]:
         if self._origin is not None and is_instance_of(
             gw, self._origin, "org.apache.spark.SparkThrowable"
         ):
-            return self._origin.getMessageParameters()
+            return dict(self._origin.getMessageParameters())
```

Comment:
this is virtually no-op. Before this change, it returns a JVM map instance that implements Python dictionary interface. After this change, it returns a pure Python dictionary.